x-pack/plugins/ingest_manager/server/saved_objects/index.ts

```diff
@@ -218,8 +219,15 @@ const savedObjectTypes: { [key: string]: SavedObjectsType } = {
           },
         },
         revision: { type: 'integer' },
+        updated_on: { type: 'keyword' },
+        updated_by: { type: 'keyword' },
+        created_on: { type: 'keyword' },
+        created_by: { type: 'keyword' },
       },
     },
+    migrations: {
```

Comment:
Even though I don't expect users to migrate from 7.8 to 7.9 I like that we get into the habit of providing a migration path! 👍